### ISSUE TITLE
Auto_increment property within DDL is generate ALTER SERIAL statement.

### ISSUE THREAD

**Issue by seunghun-kim** [CUBRID/cubrid-manager#1]
### Tables
- Database A

``` SQL
CREATE TABLE [test] (
    [col1] INTEGER NOT NULL, 
    [col2] CHARACTER VARYING(10), 
    [col3] CHARACTER VARYING(10),  
    CONSTRAINT [pk_test_tbl1_col1] PRIMARY KEY  ([col1])
) COLLATE utf8_bin;
```
- Database B

``` SQL
CREATE TABLE [test] (
    [col1] INTEGER AUTO_INCREMENT (1, 1) NOT NULL, 
    [col2] CHARACTER VARYING(10), 
    [col3] CHARACTER VARYING(10),  
    CONSTRAINT [pk_test_tbl1_col1] PRIMARY KEY  ([col1])
) COLLATE utf8_bin;
```
### Generate the DDL based B using schema comparative function in CM.

``` SQL
ALTER TABLE [test] CHANGE COLUMN col1 col1 integer AUTO_INCREMENT(1,1) NOT NULL  FIRST;
ALTER SERIAL test_ai_col1 START WITH 1 INCREMENT BY 1 MINVALUE 1;
```

**Comment by seunghun-kim** [comment]
In version at less than or equal to 8.3.1, auto_increment property will auto generated serial below format.
<tableName>_ai_<columnName>

By the way, that serial was not created because auto_increment property is not apply yet.

So, Generated DDL should be processing in accordance with the procedure below.
1. Rename the target column.
2. Add auto_increment property at new added column behind renamed column at first.
3. Update the new column sequentially.
4. Drop the renamed column at first.

For processing like that, SchemaDDL class should be refactoring because Alter statement need to separate each others.

Additionally, We need to create new table with auto_increment property and then insert all data from old table to new table because ["alter table ~ add column ~ after columnName" statement is not supported at 8.2.1.](http://www.cubrid.org/manual/821/ko/ADD%20COLUMN%20%EC%A0%88)

**Comment by seunghun-kim** [comment]
When I additional check about this issue, In all version of CUBRID, auto serial is generated when auto_increment property is exists.

As a result, This issue is not a problem at version of higher than 8.3.1.
But the problem is that `ALTER SERIAL` is generated without `ALTER DDL` at version of less than or equal 8.3.1.

Therefore, I need to change this code to generate `ALTER DDL` when version is less than or equal 8.3.1, or I should change the code to returns nothing.

In other words, `ALTER SERIAL` is not a problem.

**Comment by seunghun-kim** [comment]
I changed code for this like below.
When using version at less than or equal to 8.3.1, Popup message will be open.
![2016-11-09 15 33 32](https://cloud.githubusercontent.com/assets/16603187/20129542/d7b2cae2-a693-11e6-834b-e100a8ca8926.png)
![2016-11-09 15 31 24](https://cloud.githubusercontent.com/assets/16603187/20129550/e19a3da6-a693-11e6-80ea-9197da683856.png)

When using version at higher than 8.3.1, This function generate DDLs about ALTER TABLE and ALTER AUTO_INCREMENT.

**Comment by seunghun-kim** [comment]
I will close this issue because #16 was merged.
